### DISCUSSION TITLE
Using Pihole[Add request]

### DISCUSSION THREAD

**Discussion by meltymon** [blocklistproject/Lists#1066] [Q&A]
Hey How i Add All Lists at one on My Pihole? Or i Must be Add Every Single List?

**Comment by ghost** [comment]
By using the 'Everything' list: https://blocklistproject.github.io/Lists/everything.txt

Instructions for where to paste that link are here: https://github.com/blocklistproject/Lists#pi-hole-instructions

**Comment by gap579137** [comment]
Highly recommend only adding the lists that you want to use. The everything list is very restrictive.
